cloud-haven-react-app/src/components/UserProfile/UserProfile.js

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import { Container, Row, Col, Button } from 'react-bootstrap';
+import Accordion from 'react-bootstrap/Accordion'
+import Card from 'react-bootstrap/Card'
+import Form from 'react-bootstrap/Form'
+import profile from "../../images/profile-user.png";
+import './UserProfile.css'
+
+const UserProfile = () => {
+
+    const [ editing, setEditing ] = React.useState(false);
+    //                                                       '0' = open   null = closed
+    const [ accordionsOpen, setAccordionsOpen ] = React.useState([null, null, null]);
+
+    const toggleEditing = () => {
+        setEditing(editing ? false : true);
```

Comment:
could be abbreviated to !editing

Reply:
That's true, I copied the style of SignUp.js